prow/tide/status.go

```diff
@@ -423,7 +423,7 @@ func (sc *statusController) setStatuses(all []PullRequest, pool map[string]PullR
 			log.WithField("original-desc", original).Warn("GitHub status description needed to be truncated to fit GH API limit")
 		}
 		actualState = githubql.StatusState(strings.ToLower(string(actualState)))
-		if !sc.dontUpdateStatus.has(string(pr.Repository.Owner.Login), string(pr.Repository.Name), int(pr.Number)) && (wantState != string(actualState) || wantDesc != actualDesc) {
+		if !sc.dontUpdateStatus.has(pr.Org, pr.Repo, pr.Number) && (wantState != string(actualState) || wantDesc != actualDesc) {
```

Comment:
Does `string(pr.Repository.Owner.Login` translate to `pr.Org` here? Seems like this should be pr.AuthorLogin. Noticed this a few places below too.

Reply:
Yeah it actually does. Imo for GitHub:
- `.Repository` is the target repo, so `.Repository.Owner.Login` means the target org, in this PR `.Repository` is `kubernetes/test-infra`, and `.Repository.Owner.Login` is `kubernetes`.
- `.AuthorLogin` is the source repo, in this PR it's `chaodaiG`